WormsGifTool/GifTool/ViewModel/SelectTurnViewModel.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using GifTool.Worms;
+
+namespace GifTool.ViewModel
+{
+    internal class SelectTurnViewModel : PageViewModelBase
+    {
+        public class TurnViewModel
+        {
+            public Turn Model { get; }
+            public string Description { get; }
+
+            public TurnViewModel(Turn turn)
+            {
+                Model = turn;
+                Description = $"{turn.Team}: {string.Join(",", turn.WeaponActions.Last().Description)}";
```

Comment:
Only comment I had was that the turn list would probably be a bit more readable if the team / weapons were in different columns rather than next to each other - pretty minor though!